src/annotations/components/AnnotationsToggleButton.tsx

```diff
@@ -0,0 +1,42 @@
+// Libraries
+import React, {FC} from 'react'
+import {useSelector, useDispatch} from 'react-redux'
+
+// Selectors
+import {getAnnotationControlsVisibility} from 'src/annotations/selectors'
+
+// Actions
+import {toggleShowAnnotationsControls} from 'src/userSettings/actions'
+
+// Components
+import {Button, ComponentColor, IconFont} from '@influxdata/clockface'
+
+const AnnotationsToggleButton: FC = () => {
+  const dispatch = useDispatch()
+  const isVisible = useSelector(getAnnotationControlsVisibility)
```

Comment:
i saw that you're following the variable example, but does it make sense for the visibility of this component to exists between dashboards? same applies to variables as well, just wondering how to scope this state.

Reply:
The UX I am going for is: User turns on a stream of annotations, they are overlaid on all the graphs regardless of where the user is. So it's a global thing